src/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.snake-part {
+  height: 300px;
+  min-width: 300px;
+  width: 33.33%;
+  font-family: sans-serif;
+  color: #fff;
+  font-size: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.snake-part-1 {
+  background-color: hsl(0, 100%, 50%);
+  order: 1;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.